test/createRandomTest.cpp

```diff
@@ -106,7 +114,7 @@ int main(int argc, char *argv[])
 	read_string(s, v);
 
 	// insert new random code
-	v.get_obj().find("randomVMtest")->second.get_obj().find("pre")->second.get_obj().begin()->second.get_obj()["code"] = "0x" + randomCode;
+	v.get_obj().find("randomVMtest")->second.get_obj().find("pre")->second.get_obj().begin()->second.get_obj()["code"] = "0x" + randomCode + "55";
```

Comment:
Does it add SSTORE at the end?

Maybe it's better to create 2 codes, one without the store, one with store. It's not very random if all the test have SSTORE at the end.